DXRando/DeusEx/Classes/ClothesRack.uc

```diff
@@ -30,6 +30,9 @@ function Frob(actor Frobber, Inventory frobWith)
 
     Super.Frob(Frobber, frobWith);
 
+    foreach AllActors(class'DXRando', dxr) break;
+    if (dxr != None && dxr.flags.IsSpeedrunMode()) return;
```

Comment:
I think instead change line 52

`if (camera!=None){`

to

`if (camera!=None && !dxr.flags.IsSpeedrunMode()){`

this way you can still do a full bingo speedrun

Reply:
converted this PR to a draft